src/components/Sidebar/SimplySidebar.tsx

```
@@ -0,0 +1,117 @@
+
+  return (
+    <ProSidebar
+      id={id}
+      className={classes(void 0, className)}
+      width='280px'
+    >
+      <div className={classes('header', headerCls)}>
+        <span
+          className={classes('logo')}
+          onClick={onLogoClick}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)